CMakeLists.txt

```diff
@@ -1466,14 +1342,6 @@ add_custom_target(clean_autogenerated_files
 
 message(STATUS "building for git revision: ${ARANGODB_BUILD_REPOSITORY}")
 
-# if (VERBOSE)
```

Comment:
This looks like debugging code which might be useful one day.

Reply:
There are CMake GUI tools with which one can look at the variables set in the current environment; there is a curses variant (ccmake) and a QT variant with mouse interface and all.

I believe that would be much more helpful for debugging.